### PR TITLE
Example on how to save interpolated planes using postprocessing callback

### PR DESCRIPTION
**Pull request by svchb**

### CODE REVIEW

File: src/general/interpolation.jl [hunk 1]
```diff
@@ -59,6 +59,20 @@ function interpolate_plane_2d(min_corner, max_corner, resolution, semi, ref_syst
     return results
 end
 
+function interpolate_plane_2d(min_corner, max_corner, resolution, semi, ref_system, v, u;
+                              smoothing_length=ref_system.smoothing_length,
+                              cut_off_bnd=true)
+    sol = (; u=[(; x=(v, u))])
```
Comment (efaulhaber) [inline src/general/interpolation.jl:65]: #466 is the clean solution for this.

File: examples/fluid/dam_break_2d.jl [hunk 2]
```diff
@@ -71,6 +71,21 @@ ode = semidiscretize(semi, tspan)
...
 
+# Example on how interpolated planes can be saved using the postprocessing callback
+# function save_interpolated_plane(v, u, t, system)
+#     interpolation_start = [0.0, 0.0]
+#     interpolation_end = [tank_size[1], 3.0]
+#     resolution = 0.005
+#     file_id = Int(ceil(t*10000))
+
+#     interpolate_plane_2d_vtk(interpolation_start, interpolation_end, resolution,
+#                              semi, system, v, u, filename="plane_$file_id.vti")
```
Comment (efaulhaber) [inline examples/fluid/dam_break_2d.jl:82]: This is not correct! These have to be the unwrapped `v_ode` and `u_ode`. I realized that I forgot to send my comment in #424 that we don't just need a version with `v` and `u`, but also `v_ode`, `u_ode` and `semi`.

**Comment by svchb** [comment]
Lets merge #466 than first and I will fix this.

File: examples/fluid/dam_break_2d.jl [hunk 3]
```diff
@@ -79,7 +79,7 @@ ode = semidiscretize(semi, tspan)
 info_callback = InfoCallback(interval=100)
 
 solution_prefix = ""
-saving_callback = SolutionSavingCallback(dt=0.02, prefix=solution_prefix)
+saving_callback = SolutionSavingCallback(dt=0.1, prefix=solution_prefix)
```
Comment (efaulhaber) [inline examples/fluid/dam_break_2d.jl:82]: Why?

File: examples/fluid/dam_break_2d.jl [hunk 4]
```diff
@@ -91,14 +91,32 @@ saving_paper = SolutionSavingCallback(save_times=[0.0, 0.371, 0.584, 0.743, 1.41
 # This can be overwritten with `trixi_include`
 extra_callback = nothing
 
+# Example on how interpolated planes can be saved using the postprocessing callback.
```
Comment (efaulhaber) [inline examples/fluid/dam_break_2d.jl:94]:
```suggestion
# Example on how interpolated planes can be saved using the postprocessing callback
```

File: examples/fluid/dam_break_2d.jl [hunk 5]
```diff
@@ -91,14 +91,32 @@ saving_paper = SolutionSavingCallback(save_times=[0.0, 0.371, 0.584, 0.743, 1.41
...
+function save_interpolated_plane(v, u, t, system)
+    # Size of the patch to be interpolated
+    interpolation_start = [0.0, 0.0]
+    interpolation_end = [tank_size[1], tank_size[2]]
+
+    # The resolution the plane is interpolated to. In this case twice the original resolution.
+    resolution = 0.5 * fluid_particle_spacing
+
+    file_id = Int(ceil(t * 10000))
```
Comment (efaulhaber) [inline examples/fluid/dam_break_2d.jl:103]:
```suggestion
    file_id = ceil(Int, t * 10000)
```

**Review by efaulhaber — CHANGES_REQUESTED** [review]
Why in the `dam_break_2d.jl` example and not in `examples/postprocessing/postprocessing.jl`? That seems to be the right place for this.

**Review by efaulhaber — APPROVED** [review]

**Pull request merged**